src/components/Profile/List/index.jsx

```diff
@@ -154,17 +160,17 @@ const ListProfiles = () => {
       dataIndex: "is_current_employee",
       key: "is_current_employee",
       render: (is_current_employee) => (
-        <strong>{is_current_employee === 1 ? "YES" : "NO"}</strong>
+        <strong>{is_current_employee}</strong>
```

Comment:
remove strong tag, instaed we can add Yes with green colour text and No with red colour text with antd component

Reply:
Done!